worldedit-core/src/main/java/com/sk89q/worldedit/function/block/SnowSimulator.java

```diff
@@ -35,6 +37,7 @@
 
     private final Property<Integer> snowLayersProperty = BlockTypes.SNOW.getProperty("layers");
     private final Property<Integer> waterLevelProperty = BlockTypes.WATER.getProperty("level");
+    private final Property<String> slabTypeProperty = BlockTypes.OAK_SLAB.getProperty("type");
```

Comment:
has this been tested with slabs other than oak? the same property of two different blocks is likely a different property instance. I'm not sure that getting e.g. stone slabs' "type" property with an instance of oak slabs' property is "reliable" behavior, even if it happens to work.

Reply:
I tested this with several different slabs and it worked. Unfortunately I couldn't find something like "BlockCategories.SLABS.getProperty()".